pkg/api/routes.go

```diff
@@ -149,5 +155,11 @@ func (h *Handler) GetRoutes() []Route {
 			"/log",
 			h.SetLogLevel,
 		},
+		{
+			"GetMetrics",
+			"GET",
+			"/metrics",
```

Comment:
what is the access pattern? Prometheus will hit this route periodically? What is the refresh interval?